bpo-41887: omit leading whitespaces on ast.literal_eval (#22469)
python/cpython: Doc/library/ast.rst

   .. versionchanged:: 3.9
      Now supports creating empty sets with ``'set()'``.

   .. versionchanged:: 3.10

>       Now allows leading whitespaces

Reviewer: ```suggestion
      For string inputs, all leading whitespace characters are now omitted.
```

Reviewer 2: "For string inputs, leading whitespace is now stripped (as trailing whitespace already was)."

      Now supports creating empty sets with ``'set()'``.

   .. versionchanged:: 3.10

>       For string inputs, leading spaces and tabs is now stripped (as
>       the trailing ones already was).

Reviewer 2: ```suggestion
      For string inputs, leading spaces and tabs are now stripped.
```
